app/assets/stylesheets/components/layout.scss

```diff
@@ -189,3 +189,14 @@ i.fa {
     border-right: 1px solid $primary-colour;
   }
 }
+
+//popover css used in signup page for client side validation
+
+#signup_page .popover{
+    background-color: rgba(0,0,0,0.6);

```

Comment:
Yes. Also, move this to a separated file. It could be a `registration.scss` in the `pages` folder since you are overriding the default popover style on the sign up page.

One question I have: is it okay to override the popover style only on this page?  /cc @mssola